src/main/java/com/spring/onlinebookstore/dto/cart/UpdateCartItemDtoResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.spring.onlinebookstore.dto.cart;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+
+public record UpdateCartItemDtoResponse(
```

Comment:
Why sometimes you use classes for dto`s and sometimes records? Please be consistent and choose approach